harness/tests/experiment/utils.py

```diff
@@ -174,6 +177,61 @@ def repo_path(path: str) -> str:
     return os.path.join(os.path.dirname(__file__), "../../../", path)
 
 
+def e2e_fixtures_path(path: str) -> str:
+    # This function is a helper for testing, all necessary fixtures should be refactored instead.
+    # TODO: refactor fixtures.
```

Comment:
I'd recommend copying the fixtures you need to here, and letting the fixtures in e2e get deleted over time, rather than introduce a sneaky cross-repo dependency.

But only if we're pretty sure the fixtures in question _will_ get deleted from e2e in a reasonably short timeframe, I guess.  If they're gonna be kept for a while, probably that's bad advice...

Reply:
This function is here just for prototyping new unit tests before I refactor any fixtures. It's not actually used in the final code, and will be removed after this refactor is complete.

Reply:
Fixtures are being moved here along with their tests. This function is here to test code before I do any refactoring as a sanity check. I plan to delete it in the future along with the last refactor-related PR.